semaphore/stylelint.sh

```diff
@@ -7,6 +7,5 @@ changed=$(git diff --name-only "$fork_point" apps/site/assets/css)
 if [ -z "$changed" ]; then
   echo "No CSS files changed relative to origin/master fork point."
 else
-  gem install scss_lint -v 0.59.0 --no-document
-  scss-lint $changed
+  npx stylelint $changed --config apps/site/assets/.stylelintrc --ignore-path apps/site/assets/.stylelintignore
```

Comment:
I don't think we need this script anymore, since (thanks to your work in this PR!) all current SCSS should pass lint anyway. We could replace the whole Semaphore step that runs this script with `npm run stylelint`.

Reply:
True. Depends on whether we want to only check changed files vs linting everything every time. Figured I'd make this consistent with the credo script but we could go either way.

Reply:
The Credo script is like that is because we have a _large_ number of extant Credo failures, and all of them are "missing moduledoc" so it's impractical to clean them up all at once. I expect someday we will delete that script and replace it with `mix credo`, since there's otherwise no downside to checking all the code every time (it takes seconds).

Reply:
Point taken: I've deleted this unneeded script!